README.md

```diff
@@ -42,32 +42,138 @@ The function above will return
 }
 ```
 
-## Methods
-
-1. OK
-2. CREATED
-3. NO_CONTENT
-4. REDIRECT
-5. BAD_REQUEST
-6. UNAUTHORIZED
-7. FORBIDDEN
-8. NOT_FOUND
-9. CONFLICT
-10. SERVER_ERROR
-11. ERROR - This will auto detect which error code to send based on the message.
-
 ## API
 
-All of the methods have the same API.
+### Successful responses
+
+| Method     | Code | Description                                                                                                                 |
+| ---------- | ---- | --------------------------------------------------------------------------------------------------------------------------- |
+| OK         | 200  | Request has succeeded and the message body contains the requested information.                                              |
+| CREATED    | 201  | Request has succeeded and a new resource has been created. The message body may contain information about the new resource. |
+| NO_CONTENT | 204  | Request has succeeded but there is not content to be returned.                                                              |
```

Comment:
Request has succeeded but there is **not** content to be returned.
Should be: Request has succeeded but there is **no** content to be returned.